collections/download/index.php

```diff
@@ -132,7 +134,7 @@ function closePage(timeToClose){
 									such as substrate, associated collectors, verbatim description.
 								</div>
 								<input type="radio" name="schema" id="darwin-core" value="dwc" onclick="georefRadioClicked(this)" />
-								Darwin Core
+								<label for="darwin-core"> Darwin Core </label>
```

Comment:
Internationalize. Unless we discussed not doing it for this file (I can't remember).